source/simulator/stokes_matrix_free.cc

```diff
@@ -475,42 +476,47 @@ namespace aspect
          */
         const StokesMatrixType &stokes_matrix;
         const ABlockMatrixType &velocity_matrix;
-        const MassMatrixType &mass_matrix;
-        const PreconditionerMp                    &mp_preconditioner;
-        const PreconditionerA                     &a_preconditioner;
+        const MassMatrixType   &mass_matrix;
```

Comment:
Same here, at the moment we implicitly assume that the reader knows that the `mass_matrix` is the `Schur complement`.